src/repository/exercisesRepository.ts

```diff
@@ -325,6 +325,426 @@ export const db: Schema = {
+      Eu: 'compro',
+      Tu: 'compras',
+      'Ela/Ele/Você': 'compra',
+      Nós: 'compramos',
+      'Eles/Elas/Vocēs': 'compram',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to teach',
+      infinitive: 'ensinar',
+      Eu: 'ensino',
+      Tu: 'ensinas',
+      'Ela/Ele/Você': 'ensina',
+      Nós: 'ensinamos',
+      'Eles/Elas/Vocēs': 'ensinam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to close',
+      infinitive: 'fechar',
+      Eu: 'fecho',
+      Tu: 'fechas',
+      'Ela/Ele/Você': 'fecha',
+      Nós: 'fechamos',
+      'Eles/Elas/Vocēs': 'fecham',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to like',
+      infinitive: 'gostar',
+      Eu: 'gosto',
+      Tu: 'gostas',
+      'Ela/Ele/Você': 'gosta',
+      Nós: 'gostamos',
+      'Eles/Elas/Vocēs': 'gostam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to start',
+      infinitive: 'começar',
+      Eu: 'começo',
+      Tu: 'começas',
+      'Ela/Ele/Você': 'começa',
+      Nós: 'começamos',
+      'Eles/Elas/Vocēs': 'começam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to wash',
+      infinitive: 'lavar',
+      Eu: 'lavo',
+      Tu: 'lavas',
+      'Ela/Ele/Você': 'lava',
+      Nós: 'lavamos',
+      'Eles/Elas/Vocēs': 'lavam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to end',
+      infinitive: 'acabar',
+      Eu: 'acabo',
+      Tu: 'acabas',
+      'Ela/Ele/Você': 'acaba',
+      Nós: 'acabamos',
+      'Eles/Elas/Vocēs': 'acabam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to catch',
+      infinitive: 'apanhar',
+      Eu: 'apanho',
+      Tu: 'apanhas',
+      'Ela/Ele/Você': 'apanha',
+      Nós: 'apanhamos',
+      'Eles/Elas/Vocēs': 'apanham',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to wear',
+      infinitive: 'vestir',
+      Eu: 'visto',
+      Tu: 'vestes',
+      'Ela/Ele/Você': 'veste',
+      Nós: 'vestimos',
+      'Eles/Elas/Vocēs': 'vestem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to divide',
+      infinitive: 'dividir',
+      Eu: 'divido',
+      Tu: 'divides',
+      'Ela/Ele/Você': 'divide',
+      Nós: 'dividimos',
+      'Eles/Elas/Vocēs': 'dividem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to take off',
+      infinitive: 'despir',
+      Eu: 'dispo',
+      Tu: 'despes',
+      'Ela/Ele/Você': 'despe',
+      Nós: 'despimos',
+      'Eles/Elas/Vocēs': 'despem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to correct',
+      infinitive: 'despir',
+      Eu: 'corrijo',
+      Tu: 'corriges',
+      'Ela/Ele/Você': 'corrige',
+      Nós: 'corrigimos',
+      'Eles/Elas/Vocēs': 'corrigem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to decide',
+      infinitive: 'decidir',
+      Eu: 'decido',
+      Tu: 'decides',
+      'Ela/Ele/Você': 'decide',
+      Nós: 'decidimos',
+      'Eles/Elas/Vocēs': 'decidem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to get',
```

Comment:
is more like "to be able to do something" / "can do something"
e.g. "I can speak portuguese" - "eu **consigo** falar português"